### ISSUE TITLE
 doubt private key

### ISSUE THREAD

**Issue by gilblockchain** [brichard19/BitCrack#296]
If I have wallet.dat and get hexadecimal information, can I get the private key with this program to redeem my balance?

**Comment by ghost** [comment]
No. You can search only for private keys that are a secret number from 1 up to 2^256.
